.github/workflows/hermes.yaml

```diff
@@ -119,7 +119,115 @@ jobs:
               --workspace \
               --retries 2 \
               --test-threads=$TEST_THREADS \
-              --success-output=final
+              --success-output=final \
+              --filter-expr '!(test(modified_test_suite::))'
```

Comment:
do we really need a separate workflow test for this ?

you can just add next test here with new environment variables.

```
NEW_TRUSTING_PERIOD=30 COSMOS_REFRESH_RATE=20 \
          nix shell \
            ..#osmosis \
            ..#wasm-simapp \
            ..#rust \
            ..#cargo-nextest \
            -c \
            cargo nextest run --release \
              --workspace \
              --retries 2 \
              --test-threads=$TEST_THREADS \
              --success-output=final \
              -- modified_test_suite

```

Reply:
actually, having separate runs for unit tests is not a good practice. we need to tackle this cleanly in separate issue.

Reply:
I will open an issue then.
The ideal solution would be an `OverwriteConfig` component to allow individual tests to overwrite the necessary configurations, similar to what we had in Hermes v1

Reply:
https://github.com/informalsystems/hermes-sdk/issues/626

Reply:
for setting specific environment variables for specific tests, we can use `assay` crate.

ref: https://github.com/mgattozzi/assay?tab=readme-ov-file#env-vars